puppet/modules/lib_bendo_server/templates/bendo.exec.erb

```diff
@@ -1,4 +1,4 @@
 #!/bin/bash
 
 exec 2>&1
-exec /sbin/chpst -u app <%= @bendo_root %>/gocode/bin/bendo -storage <%= @bendo_storage_dir %> --cache-dir <%= @bendo_cache_dir %> --mysql <%= @bendo_mysql_user %>:<%= @bendo_mysql_password %>@tcp\(<%= @bendo_mysql_server %>:3306\)/<%= @bendo_mysql_db %> -user-tokens <%= @bendo_root %>/tokens
+exec /sbin/chpst -u app <%= @bendo_root %>/gocode/bin/bendo -storage <%= @bendo_storage_dir %> --cache-dir <%= @bendo_cache_dir %> <%= @bendo_cache_size %> --mysql <%= @bendo_mysql_user %>:<%= @bendo_mysql_password %>@tcp\(<%= @bendo_mysql_server %>:3306\)/<%= @bendo_mysql_db %> -user-tokens <%= @bendo_root %>/tokens
```

Comment:
Maybe put these on different lines using the bash line continuation character `\`?